helmfile.d/snippets/derived.gotmpl

```diff
@@ -32,7 +32,7 @@
 {{- $oidcBaseUrlBackchannel := "http://keycloak-http.keycloak/realms/otomi" }}
 {{- $oidcWellKnownBackchannel := printf "%s/.well-known/openid-configuration" $oidcBaseUrlBackchannel }}
 
-{{- if and (not (env "CI")) (hasKey $v.cluster "k8sContext") }}
+{{- if and (not (env "CI")) (not (env "VALUES_INPUT")) (hasKey $v.cluster "k8sContext") }}
```

Comment:
I saw that in many places `CI` has been removed but now I am reading this line and I am not sure if `CI` is the deprecated env or not. Could you clarify?

Reply:
CI is what it has always been for: to make clear the code is ran in a CI context. It was abused and now freed